ligo/README.md

```diff
@@ -28,18 +25,60 @@ which will save the compiled contract in `out/baseDAO.tz`.
 
 ## Generating `Storage`
 
-Currently we don't provide any convenient way to generate `BaseDAO` storage,
-but you can use [RegistryDAO Storage generation](#registrydao) as example.
+Defaults for the storage, including configuration, are included in the
+`ligo/src/defaults.mligo` file.
+This can be converted to a Michelson expression with the `ligo compile-storage`
+command, which is included in the Makefile so it can be run using `make`.
+
+```bash
+make admin_address="tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" token_address="tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" out/trivialDAO_storage.tz
+```
+
+Note: Both arguments are optional and will be equal to the values above if not
+specified.
+
+Note also that `make all` will generate `out/trivialDAO_storage.tz` with these
+default values as well, beside compiling the contract itself.
 
-We will work on this in future versions.
+For a more interesting example you can use the
+[RegistryDAO Storage generation](#registrydao) instead.
 
-## Using the contract
+## Originating and starting up the contract
 
 You will need [tezos-client](http://tezos.gitlab.io/introduction/howtoget.html) or similar software to originate and interact with the contract on the Tezos network.
 
 For example, if you have compiled the contract and obtained a storage (`<storage>`) as described above you can originate it with:
 `tezos-client originate contract myDAO transferring 1 from alice running out/baseDAO.tz --init <storage>`
 
+Origination, however, won't be enough to make full use of the contract.
+Due to the size limitations of Tezos, baseDAO will store its entrypoints as
+lambdas inside its storage.
+
+Because of this the contract starts in a "startup" mode and a series of operations
+is necessary to load these entrypoint lambdas into the contract, before it can
+be actually used.
+
+All the standard entrypoint lambdas are provided in `ligo/src/startup.mligo`
+(with some more information) and can be complied into parameters to push them to
+the contract with the `ligo compile-parameter` command.
+
+This is also included in the `Makefile` and using `make all` will provide each
+parameter in the `out/entrypoints` directory.
+
+For example, to load (or override) an entrypoint into the storage, you can use:
+`tezos-client transfer 0 from alice to myDAO --arg <ep_parameter>`
```

Comment:
I think we should provide a full example, since it is a bit hard to understand what needs to be done here:


Example: To load the `burn` entrypoint, run this command: 
(the byte can be found in `out/entrypoints/burn.tz`)
```
tezos-client transfer 0 from alice to myDAO --arg (Left (Some (Pair "burn"
                  (Some 0x05020000023f03210316050d0765046e000000062566726f6d5f076504620000000925746f6b656e5f696404620000000725616d6f756e74072f02000000210320074303680100000014454e545259504f494e545f4e4f545f464f554e44032702000001e0034c0317031603210316031603160316034803190325072c0200000000020000001e03200743036c030b0743036801000000094e4f545f41444d494e034203270321031603160317031605700002032105710003031703160342057000020321057100030316057000030317031703420342032103170317034c032105710002031703160570000203210571000303160317057000030316031605700003032105710004057000030321057100040570000303210571000403420329072f020000005b034c0320034c0320074303620000034c03210571000203190325072c020000000203200200000033034c0320074303620000034c03420743036801000000184641325f494e53554646494349454e545f42414c414e4345034203270200000077034c032105710002034c032105710002034b0356072f020000003b057000020320057000020320057000020320034c03420743036801000000184641325f494e53554646494349454e545f42414c414e434503420327020000001c034c0320034c032005700003034c0346057000030570000303420350034c0321057100020317057000020321057100030316031705700003032105710004031603160317031705700003034205700003031603160316034203420342053d036d0342))))
```

Reply:
I think that if we provide a full example we should use a very small custom entrypoint, because having a very long sequence of bytes isn't very useful for documentation.

Additionally I think I should add an example of calling this without fake `<ep_parameter>`, but using bash to `cat` an actual entrypoint. How does that sound?

Reply:
Yeah, I'm good either ways with using a small entrypoint or `cat`.